### PR TITLE
Add several spelling corrections and refinements

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12201,7 +12201,7 @@ overwite->overwrite
 overwites->overwrites
 overwitten->overwritten
 overwize->otherwise
-overwridden->overwritten
+overwridden->overwritten, overridden,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: I think overridden should be the first suggestion!

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -16355,6 +16355,8 @@ subsytem->subsystem
 subsytems->subsystems
 subtabels->subtables
 subtances->substances
+subtarger->subtarget
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Perhaps also add the hyphenated variants of this and the plural?

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 2982c26

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 2982c26

**Pull request merged**